src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/OpenSslCertificateAssetDownloader.cs

```diff
@@ -25,16 +25,22 @@ internal static class OpenSslCertificateAssetDownloader
             try
             {
                 X509ContentType contentType = X509Certificate2.GetCertContentType(data);
+                X509Certificate2 certificate;
+
                 switch (contentType)
                 {
                     case X509ContentType.Cert:
+                        certificate = X509CertificateLoader.LoadCertificate(data);
+                        break;
                     case X509ContentType.Pkcs7:
+#pragma warning disable SYSLIB0057 // Content is known to be PKCS7.
+                        certificate = new X509Certificate2(data);
```

Comment:
I was going to say "we could be model citizens here and load the blob with SignedCms and extract the cert the way we would tell others to do it...", but realized we have a layering problem.  So, nevermind.